engines/freescape/movement.cpp

```diff
@@ -33,65 +33,72 @@ void FreescapeEngine::initKeymaps(Common::Keymap *engineKeyMap, const char *targ
 	Common::Action *act;
 
 	act = new Common::Action(Common::kStandardActionMoveUp, _("Up"));
-	act->setKeyEvent(Common::KEYCODE_UP);
+	act->setCustomEngineActionEvent(kActionMoveUp);
+	act->addDefaultInputMapping("UP");
 	act->addDefaultInputMapping("JOY_UP");
 	act->addDefaultInputMapping("o");
+	act->allowKbdRepeats();
```

Comment:
This is not necessary and it should be removed (same as the other uses of the function). The code that handle keyboard repeats is already implemented [here](https://github.com/scummvm/scummvm/blob/master/engines/freescape/events.cpp#L33-L77) and it should not be duplicated. The reason we don't use `allowKbdRepeats` is because that depends on the used backend and the engine code should work for every backend.